### PR TITLE
Update README_cn.md

### PR DESCRIPTION
**Pull request by zhuo-yoyowz**

### CODE REVIEW

File: README_cn.md [hunk 1]
```diff
@@ -93,7 +89,7 @@ Jupyter notebooks 分为四个大类，选择一个跟你需求相关的开始
...
 | [107-speech-recognition-quantization](notebooks/107-speech-recognition-quantization/) | 优化及量化预训练Wav2Vec2语音模型 |
 | [110-ct-segmentation-quantize](notebooks/110-ct-segmentation-quantize/)<br> | 量化肾脏分割模型并展示实时推理 | 
-| [111-detection-quantization](notebooks/111-detection-quantization/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F111-detection-quantization%2F111-detection-quantization.ipynb) | 量化目标检测模型 | 
+| [111-detection-quantization](notebooks/111-detection-quantization/) | 量化目标检测模型 | 
 | [112-pytorch-post-training-quantization-nncf](notebooks/112-pytorch-post-training-quantization-nncf/) | 利用神经网络压缩框架(NNCF)在后训练模式下来量化PyTorch模型(没有模型微调)| 
 | [113-image-classification-quantization](notebooks/113-image-classification-quantization/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?labpath=notebooks%2F113-image-classification-quantization%2F113-image-classification-quantization.ipynb) | 量化mobilenet图片分类 |
```
Comment (OpenVINO-dev-contest) [inline README_cn.md:94]:
```suggestion
| [113-image-classification-quantization](notebooks/113-image-classification-quantization/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?labpath=notebooks%2F113-image-classification-quantization%2F113-image-classification-quantization.ipynb) | 量化mobilenet图片分类模型 | 
```

File: README_cn.md [hunk 2]
```diff
@@ -93,7 +89,7 @@ Jupyter notebooks 分为四个大类，选择一个跟你需求相关的开始
 | [106-auto-device](notebooks/106-auto-device/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?labpath=notebooks%2F106-auto-device%2F106-auto-device.ipynb) | 演示如何使用AUTO设备 |
 | [107-speech-recognition-quantization](notebooks/107-speech-recognition-quantization/) | 优化及量化预训练Wav2Vec2语音模型 |
 | [110-ct-segmentation-quantize](notebooks/110-ct-segmentation-quantize/)<br> | 量化肾脏分割模型并展示实时推理 | 
-| [111-detection-quantization](notebooks/111-detection-quantization/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F111-detection-quantization%2F111-detection-quantization.ipynb) | 量化目标检测模型 | 
+| [111-detection-quantization](notebooks/111-detection-quantization/) | 量化目标检测模型 | 
 | [112-pytorch-post-training-quantization-nncf](notebooks/112-pytorch-post-training-quantization-nncf/) | 利用神经网络压缩框架(NNCF)在后训练模式下来量化PyTorch模型(没有模型微调)|
```
Comment (OpenVINO-dev-contest) [inline README_cn.md:93]:
```suggestion
| [112-pytorch-post-training-quantization-nncf](notebooks/112-pytorch-post-training-quantization-nncf/) | 利用神经网络压缩框架(NNCF)在后训练模式下来量化PyTorch模型(无需模型微调)| 
```

File: README_cn.md [hunk 3]
```diff
@@ -108,10 +104,10 @@ Jupyter notebooks 分为四个大类，选择一个跟你需求相关的开始
...
-| 演示CT扫描数据的实时分割 | 对语音转文本识别模型推理 | 根据上下文回答问题 | 使用文本识别resnet对图像上的文本进行注释 | 手写体中文及日文OCR |
-|<img src="https://user-images.githubusercontent.com/15709723/134784204-cf8f7800-b84c-47f5-a1d8-25a9afab88f8.gif" width=225>| <img src="https://user-images.githubusercontent.com/36741649/140987347-279de058-55d7-4772-b013-0f2b12deaa61.png" width=225> | <img src="https://user-images.githubusercontent.com/4547501/152571639-ace628b2-e3d2-433e-8c28-9a5546d76a86.gif" width=225> | <img src="https://user-images.githubusercontent.com/36741649/129315292-a37266dc-dfb2-4749-bca5-2ac9c1e93d64.jpg" width=225> | <img width="425" alt="handwritten_simplified_chinese_test" src="https://user-images.githubusercontent.com/36741649/132660640-da2211ec-c389-450e-8980-32a75ed14abb.png"> <br> 的人不一了是他有为在责新中任自之我们 |
+| 演示CT扫描数据的实时分割 | Stable Diffusion 由文字到图片 | Whisper字幕生成 | YOLOv8优化 | InstructPix2Pix图片编辑 |
```
Comment (OpenVINO-dev-contest) [inline README_cn.md:109]:
```suggestion
| 演示CT扫描数据的实时分割 | 用Stable Diffusion由文本生成图像 | 通过OpenAI Whisper和OpenVINO进行字幕生成 | 使用NNCF PTQ API优化YOLOv8 | InstructPix2Pix图片编辑 |
```

File: README_cn.md [hunk 4]
```diff
@@ -128,23 +124,29 @@ Jupyter notebooks 分为四个大类，选择一个跟你需求相关的开始
...
+| [224-3D-segmentation-point-clouds](notebooks/224-3D-segmentation-point-clouds/)<br> |使用OpenVINO处理点云数据并进行3D分割 | <img src = "https://user-images.githubusercontent.com/91237924/185752178-3882902c-907b-4614-b0e6-ea1de08bf3ef.png" width=225> |
+| [225-stable-diffusion-text-to-image](notebooks/225-stable-diffusion-text-to-image)<br> | 用Stable Diffusion由文本生成图像 | <img src=https://user-images.githubusercontent.com/29454499/216524089-ed671fc7-a78b-42bf-aa96-9f7c791a9419.png width=225>|
+| [226-yolov7-optimization](notebooks/226-yolov7-optimization/)<br> | 使用NNCF PTQ API优化YOLOv7 | <img src=https://raw.githubusercontent.com/WongKinYiu/yolov7/main/figure/horses_prediction.jpg  width=225> |
+| [227-whisper-subtitles-generation](notebooks/227-whisper-subtitles-generation/)<br> | 利用OpenAI Whisper及OpenVINO为视频生成字幕 | <img src=https://user-images.githubusercontent.com/29454499/204548693-1304ef33-c790-490d-8a8b-d5766acb6254.png  width=225> |
+| [228-clip-zero-shot-image-classification](notebooks/228-clip-zero-shot-image-classification)<br> | 利用CLIP及 OpenVINO进行零样本图像分类 | <img src=https://user-images.githubusercontent.com/29454499/207795060-437b42f9-e801-4332-a91f-cc26471e5ba2.png  width=500> |
```
Comment (OpenVINO-dev-contest) [inline README_cn.md:146]:
```suggestion
| [228-clip-zero-shot-image-classification](notebooks/228-clip-zero-shot-image-classification)<br> | 利用CLIP及OpenVINO进行零样本图像分类 | <img src=https://user-images.githubusercontent.com/29454499/207795060-437b42f9-e801-4332-a91f-cc26471e5ba2.png  width=500> |
```

File: README_cn.md [hunk 5]
```diff
@@ -165,12 +167,31 @@ Jupyter notebooks 分为四个大类，选择一个跟你需求相关的开始
...
+| [406-3D-pose-estimation-webcam](notebooks/406-3D-pose-estimation-webcam/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks.git/main?labpath=notebooks%2F406-3D-pose-estimation-webcam%2F406-3D-pose-estimation.ipynb) | 使用网络摄像头或视频文件进行3D人体姿态估计 | <img src = "https://user-images.githubusercontent.com/42672437/183292131-576cc05a-a724-472c-8dc9-f6bc092190bf.gif" width=225> |
+| [407-person-tracking-webcam](notebooks/407-person-tracking-webcam/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F407-person-tracking-webcam%2F407-person-tracking.ipynb) | P使用网络摄像头或视频文件进行人流统计 | <img src = "https://user-images.githubusercontent.com/91237924/210479548-b70dbbaa-5948-4e49-b48e-6cb6613226da.gif" width=225> |
```
Comment (OpenVINO-dev-contest) [inline README_cn.md:190]:
```suggestion
| [407-person-tracking-webcam](notebooks/407-person-tracking-webcam/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F407-person-tracking-webcam%2F407-person-tracking.ipynb) | 使用网络摄像头或视频文件进行人体跟踪 | <img src = "https://user-images.githubusercontent.com/91237924/210479548-b70dbbaa-5948-4e49-b48e-6cb6613226da.gif" width=225> |
```

File: README_cn.md [hunk 6]
```diff
@@ -200,12 +222,14 @@ Notebooks几乎可以在任何地方运行&mdash；您的笔记本电脑、云
...
 | Windows Server 2016 or higher                              | 3.7, 3.8, 3.9, 3.10                                |
 
+
 [![-----------------------------------------------------](https://user-images.githubusercontent.com/10940214/155750931-fc094349-b6ec-4e1f-9f9a-113e67941119.jpg)](#)
 <div id='-run-the-notebooks'/>
 
 ## 💻 运行Notebooks
 
 ### 启动单个Notebook
+### To Launch a Single Notebook
```
Comment (OpenVINO-dev-contest) [inline README_cn.md:232]:
```suggestion
```

File: README_cn.md [hunk 7]
```diff
@@ -214,6 +238,7 @@ jupyter 201-vision-monodepth.ipynb
 ```
 
 ### 启动所有Notebooks
+### To Launch all Notebooks
```
Comment (OpenVINO-dev-contest) [inline README_cn.md:241]:
```suggestion
```

**Review by OpenVINO-dev-contest — CHANGES_REQUESTED** [review]
Hi @zhuo-yoyowz Thanks for your update, could you also help to add 117-model-server into this README. It was merged yesterday, great thanks.

**Comment by zhuo-yoyowz** [comment]
> Hi @zhuo-yoyowz Thanks for your update, could you also help to add 117-model-server into this README. It was merged yesterday, great thanks.

Added. Thanks~

**Review by OpenVINO-dev-contest — APPROVED** [review]

**Review by eaidova — APPROVED** [review]

**Comment by eaidova** [comment]
@zhuo-yoyowz @OpenVINO-dev-contest  thank you for update! Great work!

**Pull request merged**